docs/userGuide/syntax/pageLayouts.mbdf

```diff
@@ -1,5 +1,9 @@
+<!---
+Since we are using code blocks, we need to be able to escape the variable in the code block in order to render it as is rather than replace it with the actual value.
+TODO: Change to a proper escape tag once PR#1049 is ready. 
+-->
 <variable name="mainContentBody">
-<code>{<span></span>{ MAIN_CONTENT_BODY }}</code>
+{&#8203;{ MAIN_CONTENT_BODY }}
```

Comment:
Ah I didn't realise this earlier, but the zero width space is preserved when the user copies the text from the page. This would cause the snippet to not work if the user copies and pastes it into their own editor. 

Let's revert back to the original version for the segments that require this variable, and leave a todo for us to change it once the escape tags PR is ready.

Reply:
Done. We should be good to go, I guess.